public/Lab-Files/Lesson02/Challenges/text-level-elements.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Text-Level-Elements</title>
+</head>
+<body>
+      
+    <h1 style="text-align:center">Text Level Elements</h1>
+     
+    <p style="text-align:left"><em>Web development</em> is a broad term for the work involved in developing a 
+    web site for the <strong>Internet</strong> (World Wide Web) or an <strong>intranet</strong> (a private network). 
+    Web development can range from developing the simplest static single 
+    page of plain text to the most complex web-based internet applications,
+    electronic businesses, and social network services. A more comprehensive
+    list of tasks to which web development commonly refers, may include web
+    engineering, web design, web content development, client liaison, 
+    client-side/server-side scripting, web server and network security 
+    configuration, and e-commerce development. Among web professionals, 
+    <strong>"web development"</strong> usually refers to the main non-design aspects of 
+    building web sites: writing markup and coding. Most recently Web 
+    development has come to mean the creation of <em>content management 
+    systems</em> or <strong>CMS</strong>. These CMS can be made from scratch, proprietary 
+    or open source. In broad terms the CMS acts as middleware between the 
+    database and the user through the browser. A principle benefit of a CMS 
+    is that it allows non-technical people to make changes to their web 
+    site without having technical knowledge.</p>
+    
+</body>
+</html>
```

Comment:
Overall you did an amazing job with the challenges. All of your code was validated, you went above and beyond and I know you worked hard. If you keep it up, I know you will succeed.